modules/creating-an-infra-node.adoc

```diff
@@ -62,10 +64,9 @@ apiVersion: config.openshift.io/v1
 kind: Scheduler
 metadata:
   name: cluster
-...
 spec:
   defaultNodeSelector: topology.kubernetes.io/region=us-east-1 <1>
-...
+#...
```

Comment:
I believe the updated [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting-1) have it as:

```suggestion
# ...
```
with a space. So this change, throughout.